main.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Linters are helpful in ensuring that we have clean code and use the best practices so kindly remove this disable and please try to fix the error instead.